__tests__/app.test.js

```diff
@@ -19,8 +20,11 @@ describe("GET /api/topics", () => {
       .then((response) => {
         expect(response.status).toBe(200);
         expect(Array.isArray(response.body.topic)).toBe(true);
-        expect(response.body.topic.length).toBeGreaterThan(0);
-        expect(typeof response.body.topic[0]).toBe("object");
+        expect(response.body.topic.length).toBe(3);
+        const allObjects = response.body.topic.every((element) => {
+          return typeof element === "object";
+        });
+        expect(allObjects).toBe(true);
```

Comment:
We need to be more specific with what we are testing here. 

You are checking how many objects are coming back which is great - ✅ 

You are checking that everything contained in the array is an object but we are missing out on being more specific and checking the actual shape of the object that is coming back. What properties are we expecting? This needs to be changed to reflect testing for the shape of the object and the properties that it contains.

Reply:
A further point to add to this - this would also pass if you send back an Array of nested Arrays as the data type of an Array in JS is 'object'!